### PR TITLE
fixed: #379 new group input focus-out handler

### LINKED ISSUE #379: new group directory disappear after focusout
In MacOS app when you want to create a new group if after enter group name you focus out from input to somewhere else, group is not creating and group be disappear.

I think this is a UX issue and user should be able submit a group with focus out by tab, click or whatever it could.

### PR DESCRIPTION
**Pull request by snhasani**
Fixes #379

### CODE REVIEW

**Comment by perry-mitchell** [comment]
Thanks @snhasani - I'll let @sallar review. I think it'd be best to hold this for a larger release once merged.

File: src/renderer/components/tree-view/tree-label-edit.js [hunk 1]
```diff
@@ -19,20 +19,25 @@ export default class LabelEditor extends Component {
   };
 
   state = {
-    title: 'Untitled'
+    title: 'Untitle'
```
Comment (sallar) [inline src/renderer/components/tree-view/tree-label-edit.js]: Why is **Untitled** changed to **Untitle**?

File: src/renderer/components/tree-view/tree-label-edit.js [hunk 2]
```diff
@@ -19,20 +19,25 @@ export default class LabelEditor extends Component {
...
   };
 
-  handleBlur() {
-    this.props.onDismiss();
+  _handleChange(e) {
```
Comment (sallar) [inline src/renderer/components/tree-view/tree-label-edit.js]: Is there a reason why all the handlers now have `_` at the beginning? I know it makes private methods obvious but let's keep the repo's convention for now, what do you think?

File: src/renderer/components/tree-view/tree-label-edit.js (hunk 1 above)
Comment (snhasani) [reply src/renderer/components/tree-view/tree-label-edit.js]: oh, this is a typo! I fix it.

File: src/renderer/components/tree-view/tree-label-edit.js (hunk 2 above)
Comment (snhasani) [reply src/renderer/components/tree-view/tree-label-edit.js]: I think they help to code readability. I am okay with naming without underline, also repo's convention is important anyway. I remove those underlines.

**Pull request merged**